suite/tests/CMakeLists.txt

```diff
@@ -1459,6 +1459,18 @@ function(torun test key source native standalone_dr dr_ops exe_ops added_out)
     # global reg keys and can thus run multiple instances of the same
     # app name simultaneously so we do not need to mark RUN_SERIAL
   endif (WIN32)
+
+  # A subset of release tests has been added and labeled with RUN_IN_RELEASE. It focuses
+  # on Google's use of the drcachesim client. We do not want to extend the Travis
+  # regression time by too much and keep the following list concise.
+  # amhere
+  if (NOT DEBUG AND UNIX AND X64 AND NOT CMAKE_COMPILER_IS_CLANG)
+    if (${test} MATCHES "common" OR
+        ${test} MATCHES "drcachesim" OR
+        ${test} MATCHES "drcacheoff")
```

Comment:
OK, I was originally thinking of a per-individual-test attribute, making it easy to add a single test at the test creation site, but I suppose this works for our current purposes.

Reply:
(No action required -- wish github had a checkbox for that.)

Reply:
You can set the property until after the tests have been added. So this seemed the best place, unless we want to add an parameter that's passed into the torun_ functions.. Ok, thanks.

Reply:
Like I said, here and in the prior PR comments, "a per-individual-test attribute" which is *ours*.  Just like all the other properties we pass to here using variables named test_foo.

Reply:
E.g., the _depends attribute, the _dr_ops attribute, the _env attribute, etc.